### PR TITLE
New report frames.

### PR DESCRIPTION
**Pull request by gislenius**

### CODE REVIEW

**Comment by gislenius** [comment]
@wcandillon could you look into the process out of memory problem? Thanks!

**Comment by wcandillon** [comment]
I'm currently looking at it... more tomorrow.

**Comment by wcandillon** [comment]
@gislenius UpdateReportSchema.jq is 22MB, maybe we over did this time? :)

**Comment by wcandillon** [comment]
@dknochen Maybe we need to discuss this offline how we can fix this.

**Comment by wcandillon** [comment]
@gislenius @dknochen In the meantime, should we by pass the xqlint check of this file? I think it's fair since it is a generated file. For javascript we never lint generated files.

**Comment by dknochen** [comment]
@wcandillon I deacitvated it for now. Sorry, for the accidental commit to master :sob: ...

**Comment by dknochen** [comment]
I also added an issue to improve: https://github.com/28msec/cellstore/issues/872

**Comment by wcandillon** [comment]
@gislenius How files like COMID-BSC-CF1-ISM-IEMIX-OILY.json come from? How are they generated?

File: tasks/lint.js [hunk 1]
```diff
@@ -31,9 +31,14 @@ gulp.task('lint:jsonlint', function(){
 
 gulp.task('lint:xqlint', function(done){
 
-    var files = _.reduce(Config.paths.jsoniq, function(result, path) {
+    var files = _.chain(Config.paths.jsoniq)
+        .reduce(function(result, path) {
                     return _.union(result, expand(path));
-                }, []);
+                }, [])
+        .filter(function(path){
```
Comment (wcandillon) [inline tasks/lint.js]: Why is it using a different method than in line 21?

Comment (wcandillon) [inline tasks/lint.js]: Yes that was a silly question, my apologies!

**Pull request merged**